core/src/main/scala/org/http4s/headers/Forwarded.scala

```diff
@@ -193,7 +193,7 @@ object Forwarded
       ParseFailure("missing host", s"no host defined in the URI '$uri'")
   }
 
-  override def parse(s: String): ParseResult[Forwarded] = parser.HttpHeaderParser.FORWARDED(s)
+  override def parse(s: String): ParseResult[Forwarded] = ???
```

Comment:
I deleted this somewhere along the way and reverted. @fredshonhorio is looking at the Forwarded header.